pkg/reconciler/cloudeventingress/resources/cloudeventingress.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/knative/eventing-sources/pkg/apis/sources/v1alpha1"
+	v1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CloudEventIngressArgs are the arguments needed to create a CloudEvent Ingress. Every
+// field is required.
+type CloudEventIngressArgs struct {
+	Image   string
+	Source  *v1alpha1.CronJobSource
```

Comment:
Should be `CloudEventIngress`, I believe.

Reply:
Changed. Please review the new version.